.github/workflows/trigger_ci.yml

```diff
@@ -0,0 +1,20 @@
+on:
+  pull_request:
+jobs:
+  mirror_repo:
+    environment: GITLAB
+    runs-on: self-hosted
+    steps:
+      - name: Mirror Repository
```

Comment:
The repository is already mirrored, right? Is this "updating" the mirror or forcing a sync?

Reply:
yes, this enforces the mirroring to be done immediately

Reply:
Can we call it "Sync Mirror Repository" then? Sorry for the nits, just want it to be crystal clear.

Reply:
I'm trying to understand the intention. 
CI can be triggered without this change, isn't?

Reply:
Not automatically from github side. Nor would it report back a pipeline on the github PR status checks without this change

Reply:
> Not automatically from github side. Nor would it report back a pipeline on the github PR status checks without the corresponding gitlab PR's changes
That's right.